src/Tools/CommonNetCoreReferences/project.json

```diff
@@ -34,6 +34,8 @@
         "System.Runtime.InteropServices": "4.3.0",
         "System.Runtime.Loader": "4.3.0",
         "System.Runtime.Numerics": "4.3.0",
+        "System.Runtime.Serialization.Primitives": "4.3.0",
+        "System.Runtime.Serialization.Json": "4.3.0",
```

Comment:
Without these using `csi` to run the BuildNuGets file was not working when hosted on core.